actionpack/lib/action_controller/test_case.rb

```diff
@@ -491,58 +491,64 @@ def determine_default_controller_class(name)
         end
       end
 
-      # Simulate a GET request with the given parameters.
+      # Simulate a GET request with the given keyword arguments.
```

Comment:
"parameters" is maybe a better fit here ; "keyword arguments" is more an implementation detail and we are still passing the action as a normal parameter, not as a keyword argument. What do you think ?